packages/components/modules/messages/GroupDetails/GroupDetailsHeader/types.ts

```
@@ -0,0 +1,5 @@
+export interface GroupDetailsHeaderProps {
+  onBackButtonClicked: () => {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)